_opensearch/java-rest-high-level.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: Java high-level REST client
+nav_order: 97
+---
+
+# Java high-level REST client
+
+The Elasticsearch OSS Java high-level REST client allows you to interact with your OpenSearch clusters and indices through request objects and response objects. To create an index, add data to a document, or complete some other operation, you submit request objects to your cluster, and you get back response objects that have all of the available information, such as the associated index or ID, returned from your cluster.
```

Comment:
The big appeal here is really that you can use Java methods/objects rather than having to manually specify the path, HTTP method, and raw JSON. It's just a nicer way to interact with the API.